vulkan_generator/BUILD.bazel

```diff
@@ -46,3 +47,22 @@ py_lint(
     name = "lint",
     srcs = glob(["*.py"]),
 )
+
+basic_vulkan_generator(
+    name = "handle_remapper_src",
+    target = "HandleRemapper"
+)
+
+cc_library(
+   name = "handle_remapper",
+   srcs = glob([":handle_remapper_src"], exclude = ["*_test.cpp",],),
+   visibility = ["//visibility:public"],
+)
+
+cc_test(
+    name = "handle_remapper_test",
+    srcs = [":handle_remapper_src"],
+    deps = [
+        "@com_google_googletest//:gtest_main",
+    ],
+)
```

Comment:
These rules should go where the replayer's code will live. Time for a new top-level directory named `replayer`?

Reply:
For now there is no replayer, so no place to put the files. But once there is one, I agree